packages/components/psammead-grid/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-grid",
-  "version": "1.0.0-alpha.15",
+  "version": "1.0.0-alpha.16",
```

Comment:
I'm not clear on how version bumps should be handled for alpha components, IMO this should be a `major` change

Reply:
True, since it's pre-version 1.0.0, I'll make it a `1.1.0-alpha.0`.